WireAPI/Sources/WireAPI/Network/NetworkService/NetworkService.swift

```diff
@@ -122,27 +124,23 @@ extension NetworkService: URLSessionDataDelegate {
     public func urlSession(
```

Comment:
I've updated this to use the `async` version of the delegate method as it needs to use `async` code.